### PR TITLE
Register MsgPackRpcServer

### PR DESCRIPTION
**Pull request by remore**

### CODE REVIEW

File: MsgPackRpcServer/url [hunk 1]
```diff
@@ -0,0 +1 @@
+git://github.com/remore/MsgPackRpcServer.git
```
Comment (tkelman) [inline MsgPackRpcServer/url]: repository name should end in .jl

**Comment by remore** [comment]
@tkelman thank you for the reveiw. I have just fix my github repo name and MsgPackRpcServer/url file, but sha1 is still the same.

Would this change be enough?

**Comment by tkelman** [comment]
Yes, that's fine. If you enabled Travis CI, would your tests pass there?

**Comment by remore** [comment]
Thanks!

I have just finished to configure Travis CI - the build status is passing now.
https://github.com/remore/MsgPackRpcServer.jl

**Pull request merged**